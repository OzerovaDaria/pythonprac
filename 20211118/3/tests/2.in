a, b = Alpha(c = 2, z = 0, a = 3), Alpha(c = 0, z = 7, a = 1)
print(a)
print(b)
