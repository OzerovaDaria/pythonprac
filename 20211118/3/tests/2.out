a: 3, c: 2, z: 0
a: 1, c: 0, z: 7
